### PR TITLE
Templated stability summary to use data from yaml

### PR DESCRIPTION
**Pull request by dpshelio**
Just to make updates simpler...

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -183,6 +194,20 @@
...
 
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    # Make sure we're outputting HTML
+    if app.builder.format != 'html':
+        return
+    src = source[0]
+    if "Current status" in src[:20]:
```
Comment (dpshelio) [inline docs/conf.py]: Yeah! this is very very hacky!!! (that's how the stability file starts..., we could use some keyword/flag so we can render more files than just this one!)

File: docs/stability.rst [hunk 2]
```diff
@@ -80,237 +84,19 @@ The current planned and existing sub-packages are:
                 Comments
             </th>
         </tr>
+    {% for module, prop in sunpy_modules.items() %}
         <tr>
             <td>
-                sunpy.cm
+                <a href="code_ref/{{ module }}.html">sunpy.{{ module }}</a>
```
Comment (dpshelio) [inline docs/stability.rst]: I tried multiple way to get the `ref` links, but it seems it doesn't work inside `..raw::` or am I doing something wrong?

File: docs/sunpy_stability.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+cm:
+  dev: dev
```
Comment (dpshelio) [inline docs/sunpy_stability.yaml]: maybe instead of `dev: xxx` we could call that `status: xxx` - I've just thought about it... but I'm going to bed now.

Comment (nabobalis) [reply docs/sunpy_stability.yaml]: Yeah, I will rename this after I merge this.

**Pull request merged**